libs/dh/shared/feature-authorization/src/lib/dh-selected-actor.component.html

```diff
@@ -15,10 +15,65 @@
 limitations under the License.
 -->
 <ng-container *rxLet="selectedActor$ as selectedActor">
-  <div class="selected-actor-container" *ngIf="!(isLoading$ | push)">
-    <div class="watt-label organization-name-label">
-      {{ selectedActor.organizationName }}
+  <!-- there's a bug when *ngIf is used on an element that acts as an overview trigger, hence the *ngIf in seperate ng-container -->
+  <ng-container *ngIf="!(isLoading$ | push)">
+    <div
+      class="selected-actor-container"
+      (click)="isOpen = !isOpen"
+      cdkOverlayOrigin
+      #trigger="cdkOverlayOrigin"
+    >
+      <div class="watt-label selected-organization-name-label">
+        {{ selectedActor.organizationName }}
+      </div>
+      <div class="watt-label selected-gln-label">{{ selectedActor.gln }}</div>
+      <div
+        class="watt-label selected-actor-name-label"
+        *ngIf="selectedActor.actorName"
+      >
+        {{ selectedActor.actorName }}
+      </div>
     </div>
-    <div class="watt-label gln-label">{{ selectedActor.gln }}</div>
-  </div>
+    <ng-template
```

Comment:
suggestion: let's move the `ng-template` outside of the `ng-container`s to avoid unnecessary nesting.

Reply:
it seems it needs to be at the same level as the trigger component 🤔